src/lightning/pytorch/trainer/connectors/accelerator_connector.py

```diff
@@ -408,13 +408,16 @@ def _choose_and_init_cluster_environment(self) -> ClusterEnvironment:
         return LightningEnvironment()
 
     def _choose_strategy(self) -> Union[Strategy, str]:
```

Comment:
Can we refactor it into this logic? I think that would be a bit cleaner.

```py
if _habana_available_and_importable():
    from lightning_habana import HPUAccelerator

    if self._accelerator_flag == "hpu" or isinstance(self._accelerator_flag, HPUAccelerator):
        # select strategy

if self._accelerator_flag == "hpu" and not _habana_available_and_importable():
	raise ImportError(...)
```

Reply:
yes, I was avoiding a top level import. This looks cleaner. Should also avoid the current pipeline fail.